### ISSUE TITLE
[3.3.5] Watch Commander Zalaphil uses wrong equipment

### ISSUE THREAD

**Issue by k59** [TrinityCore/TrinityCore#14636]
67ce9924a2aa TDB 335.58
---

*Watch Commander Zalaphil (ID: 5809)* is using the equipment for his post-Cataclysm version in 3.3.5.  

First attached picture shows the NPC featured in TDB(335.58) and the second shows how he's supposed to look prior to Cataclysm.

![TDB 335.58](http://i.imgur.com/dSksv4p.png)

![Pre-Cataclysm Reference](http://vignette3.wikia.nocookie.net/wowwiki/images/c/ce/Watch_Commander_Zalaphil.jpg/revision/latest?cb=20100310060850)

**Comment by Killyana** [comment]
``` sql
UPDATE `creature_equip_template` SET `itemEntry1`=2147, `itemEntry2`=2053 WHERE `entry`=5809;

**Comment by k59** [comment]
Can confirm that the patch solved the issue.

![Killyana fix](http://i.imgur.com/306t2pb.png)

**Comment by Kinzcool** [comment]
The ID he actually uses is right for cata+, so be wary when adding this "fix" as it is only for 3.3.5a.
